### PR TITLE
DOCSP-30203 Suggested Mapping Rules Are Previewed

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## Description

Adding visual to mapping rules stub page for the blue box preview functionality. Also adding a little detail to the note directive for suggested mapping rules.

## Staging

https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-30203/mapping-rules/mapping-rules/#preview-mapping-rules

https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-30203/mapping-rules/new-rules-suggested-mappings/#steps

## Jira

https://jira.mongodb.org/browse/DOCSP-30203

## Build

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64aec475423952aeb98e7d78

### CODE REVIEW

File: source/mapping-rules/new-rules-suggested-mappings.txt [hunk 1]
```diff
@@ -56,6 +56,12 @@ Steps
       These mapping rules are not added to your project until you click the 
       :guilabel:`Apply` button.
 
+      - If the collection entity does not exist, it is added to the
```
Comment (jwilliams-mongo) [inline source/mapping-rules/new-rules-suggested-mappings.txt:59]:
```suggestion
      - If the collection entity doesn't exist, it is added to the 
```

**Review by jwilliams-mongo — APPROVED** [review]
@ianf-mongodb LGTM % two minor nits

**Comment by ianf-mongodb** [comment]
Hi @alwaysbemark, Tom is OOO, would you mind taking a look at this update for LGTM approval please?

File: source/mapping-rules/new-rules-suggested-mappings.txt [hunk 2]
```diff
@@ -56,6 +56,12 @@ Steps
       These mapping rules are not added to your project until you click the 
       :guilabel:`Apply` button.
 
+      - If the collection entity doesn't exist, it is added to the
```
Comment (alwaysbemark) [inline source/mapping-rules/new-rules-suggested-mappings.txt:59]: I think we only allow suggesting from existing collections. What is the action being desscribed here?

File: source/includes/mapping-options-preview.rst [hunk 3]
```diff
@@ -1,6 +1,12 @@
-The blue dotted box in the MongoDB diagram pane is a visual preview of how each 
-rule and option affect your destination data model. Using this feature, 
-you can preview your destination data model before you run a sync job.
+You can preview how mapping rules affect your data model by viewing the 
+blue dotted box in the MongoDB diagram pane. This allows you to preview 
+the mapping rule before you run a sync job.
```
Comment (alwaysbemark) [inline source/includes/mapping-options-preview.rst:3]: Maybe `before you apply it` as there are quite a few steps in between creating the mapping and running a job?

File: source/index.txt [hunk 4]
```diff
@@ -19,7 +19,7 @@ What is Relational Migrator
    |
 
    .. button:: Download Relational Migrator
-      :uri: http://mongodb.com/try/download/relational-migrator
+      :uri: https://mongodb.com/try/download/relational-migrator
```
Comment (ianf-mongodb) [inline source/index.txt:22]: You can ignore this, hopefully it fixes the external link icon on the download button.

File: source/mapping-rules/new-rules-suggested-mappings.txt (hunk 2 above)
Comment (ianf-mongodb) [reply source/mapping-rules/new-rules-suggested-mappings.txt]: On reading this again, I think it makes more sense to remove both bullet points. I think the original note directive covers this effectively.

File: source/includes/mapping-options-preview.rst (hunk 3 above)
Comment (ianf-mongodb) [reply source/includes/mapping-options-preview.rst]: Sure that makes sense, updated.

**Review by ianf-mongodb — COMMENTED** [review]
@alwaysbemark Feedback has been implemented!

**Review by alwaysbemark — APPROVED** [review]
Looks good, thanks Ian!

**Pull request merged**